examples/src/letter_a.rs

```diff
@@ -0,0 +1,59 @@
+use glam::DVec3;
+use opencascade::primitives::{Edge, Face, IntoShape, Shape, Wire};
+
+type Contour = Vec<Vec<(i32, i32)>>;
+
+pub fn shape() -> Shape {
+    let outer = contour_to_face(outer_contour()).extrude(DVec3::new(0.0, 0.0, 10.0));
+    let inner = contour_to_face(inner_contour()).extrude(DVec3::new(0.0, 0.0, 10.0));
+    outer.subtract(&inner).into_shape()
+}
+
+fn contour_to_face(contour: Contour) -> Face {
+    let edges: Vec<Edge> = contour
+        .into_iter()
+        .map(|segment_points| {
+            let points = segment_points
+                .into_iter()
+                .map(|(x, y)| DVec3::new(x as f64 / 16.0 - 16.0, y as f64 / 16.0 - 16.0, 0.0));
```

Comment:
Can you explain the `/ 16.0 - 16.0` ? Is this some sort of font-specific thing?

Reply:
Oh yeah, those are totally "magic numbers"... The contour data used in the example is dumped straight from the TTF file. The `16` is a constant I determined via Eyeball Mark I (TM) to scale the resulting shape to a reasonable size in the viewer, and also more-or-less center it.

Reply:
Could bake those constants into the raw outline data if you prefer that.

Reply:
Yeah either baking it or at least pulling them out into named constants would be good.

Reply:
Done! :)